src/tests/tests/htmlutils.js

```diff
@@ -72,32 +47,33 @@ QUnit.test("getActionDescription", (assert) => {
 
 QUnit.test("getToggleHtml", function (assert) {
   // Test parameters
-  var tests = [
+  const origin = "pbtest.org";
+  const tests = [
     {
-      origin: "pbtest.org",
-      action: "block",
-      expectedResult: "0",
+      action: constants.BLOCK,
+      expectedResult: constants.BLOCK,
     },
     {
-      origin: "pbtest.org",
-      action: "cookieblock",
-      expectedResult: "1",
+      action: constants.COOKIEBLOCK,
+      expectedResult: constants.COOKIEBLOCK,
     },
     {
-      origin: "pbtest.org",
-      action: "allow",
-      expectedResult: "2",
+      action: constants.ALLOW,
+      expectedResult: constants.ALLOW,
+    },
+    {
+      action: constants.DNT,
+      expectedResult: constants.ALLOW,
     },
   ];
 
   // Run each test.
-  for (var i = 0; i < tests.length; i++) {
-    var origin = tests[i].origin;
-    var action = tests[i].action;
-    var expected = tests[i].expectedResult;
-    var message = "Inputs: '" + origin + "' and '" + action + "'";
-    var html = htmlUtils.getToggleHtml(origin, action);
-    var inputValue = $('input[name="' + origin + '"]:checked', html).val();
+  for (let i = 0; i < tests.length; i++) {
```

Comment:
AFAIK there isn't a reason to use old school ES5 for loops anymore. A `forEach` or `for ... of` ought to work the same here, as well as making it more readable